test/integration/multikueue/suite_test.go

```diff
@@ -89,13 +89,15 @@ func TestMultiKueue(t *testing.T) {
 }
 
 func createCluster(setupFnc framework.ManagerSetup, apiFeatureGates ...string) cluster {
+	projectDirectory := os.Getenv("PROJECT_DIR")
```

Comment:
This looks great, but it is a little bit off the main purpose of the pr. Would you mind extracting this change to separate preparatory PR?